packages/contracts/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "0x-smart-contracts",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "transpile": "rm -rf ./transpiled; copyfiles ./build/**/* ./deploy/solc/solc_bin/* ./deploy/test/fixtures/contracts/**/* ./deploy/test/fixtures/contracts/* ./transpiled; tsc;",
+    "test": "npm run transpile; truffle test",
+    "compile": "npm run transpile; node transpiled/deploy/cli.js compile",
+    "migrate": "npm run transpile; truffle migrate",
+    "migrateCustom": "npm run transpile; node transpiled/deploy/cli.js migrate",
+    "deploy": "npm run migrate --network kovan",
+    "lint": "tslint --project . 'migrations/*.ts' 'test/**/*.ts' 'util/*.ts' 'deploy/**/*.ts'",
+    "testrpc": "testrpc --networkId 50",
```

Comment:
Let's remove this. We now have a `testrpc` command in the top-level `package.json`. The developer only needs one such command for all the sub-packages. 

Remember to remove the `testrpc` dependency too!